src/lib/imports.ts

```diff
@@ -0,0 +1,40 @@
+export function getImportInsertion(
```

Comment:
Deals with detecting existing imports, both of style files as well as `styled` import.

Returns the minimal text we need to change, and where in the file we need to perform the change.